.scala-steward.conf

```diff
@@ -1,17 +0,0 @@
-updates.ignore = [
-  { groupId = "com.google.api", artifactId = "gax" },
-  { groupId = "com.google.api", artifactId = "gax-grpc" },
-  { groupId = "org.apache.httpcomponents", artifactId = "httpcore" },
-  { groupId = "io.grpc" },
```

Comment:
hmm, I'm not fully sold on replacing the entries that are just a groupId (such as `io.grpc`) with `build.sbt` declarations... it seems like it adds verbosity in the build file because you have to enumerate all the affected artifacts. What does the rest of the team think? @kellen / @jto ?

Reply:
```
// scala-steward:off
// 50 lines
// scala-steward:on
```
leaves a little to be desired in terms of clarity IMO. Can we put the beam-managed versions in a separate file instead?

Reply:
I think it will be confusing if we only move the versions to another file.

Maybe we can start by wrapping version into an object, and move the full dependency declaration to another file in a later step

Reply:
Hm only noticed this now; hm I don't really like the fact that we are now coupling scala-steward logic with the build, it just adds complexity. Plus as we are seeing this PR added back the `.scala-steward.conf` so really there's not much point in keeping things in both places.

Reply:
Not sure if it would be that much more tedious tbh; for most cases we can actually ignore the hole group; But even if it's a little bit more effort the fact that it's decoupled from the build def and in one since place makes it easier to reason about things.